run.sh

```diff
@@ -116,8 +119,12 @@ main() {
 
   h3 "Adjusting file permissions"
   groupadd -f docker && usermod -aG docker www-data
-  find /app -type d -exec chmod 755 {} \;
-  find /app -type f -exec chmod 644 {} \;
+  if [[ "${PHP_ENV}" =~ ^dev ]]; then
+    chmod -R 777 /app
```

Comment:
The canonical user for this would be `www-data`. I can fix this when I get some free time in the next week or so.